src/webserver/webserver.go

```diff
@@ -0,0 +1,12 @@
+package webserver
+
+import (
+	"net/http"
+)
+
+func Server(port int) {
+	http.HandleFunc("/new", New)
+	http.HandleFunc("/move", Move)
```

Comment:
You should restrict routes using correct HTTP actions. Here for instance, you need a `POST` only route.